src/App.tsx

```diff
@@ -1,24 +1,16 @@
 import React, { FC } from 'react';
```

Comment:
I suggest formating the whole file.

Reply:
Resolved on 6a889336287b010958f6a9be02002bb9c7ff4b17